.github/workflows/upload-pypi-source.yml

```diff
@@ -49,6 +49,14 @@ jobs:
         python -m pip install --upgrade pip setuptools wheel
         python -m pip install .[dev]
 
+    - name: Lint source with flake8
```

Comment:
do we need these here? They are ran by superlinter/pre-commit already

Reply:
Shouldn't that be a separate discussion from PAC?

Reply:
And specifically, mypy is run most correctly here.

Reply:
I guess, lets merge this